src/signals/incident-management/containers/Dashboard/components/Filter/index.tsx

```diff
@@ -0,0 +1,3 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
```

Comment:
Usually we don't add the SPDX to index files.

Reply:
> Looking good! Do have some notes though:
> 
> * PR title
> * In the design there is no filter for Departments. Why is it added here?
> * There is a todo open from yourself

when there are multiple departments, you have to select one of them